lang/hr/index.md

```diff
@@ -0,0 +1,262 @@
+---
+title: Semantičko verzioniranje 2.0.0
+language: hr
+---
+
+Semantičko verzioniranje 2.0.0
+==============================
+
+Sažetak
+-------
+
+Uzevši oznake verzije VEĆA.MANJA.ZAKRPA, povećavajte:
+
+1. VEĆU verziju kada napravite inkompatibilne promjene u APIju,
+1. MANJU verziju kada dodate unatrag-kompatibilne funkcionalnosti i
+1. ZAKRPA verziju kada napravite unatrag-kompatibilne ispravke bugova.
+
+Dodatne oznake za pred-izdanja i metapodatke builda mogu se koristiti kao proširenje na VEĆA.MANJA.ZAKRPA format.
```

Comment:
Maybe instead of "builda" put "programskog paketa", "programskog proizvoda", "programske isporuke". I am not completely sure about a Croatian equivalent for a build.

Reply:
I would rather leave it and put it in italics or something - I'm sure developers understand it and use it, and any attempt at translating might just confuse people. As you said, there probably isn't a good equivalent. "programskog paketa", "programskog proizvoda", "programske isporuke" translate closer *package*, *product* and *release*.